rakefile.rb

```diff
@@ -5,20 +5,34 @@
   raise "MSBuild not found"
 end
 
-nuget_command  = ".nuget/nuget.exe"
-nunit_command  = "packages/NUnit.Runners.2.6.3/tools/nunit-console.exe"
-xunit_command = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
+nuget_command   = ".nuget/nuget.exe"
+gitlink_command = "packages/gitlink.2.2.0/lib/net45/GitLink.exe"
+nunit_command   = "packages/NUnit.Runners.2.6.3/tools/nunit-console.exe"
+xunit_command   = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
 
 solution        = "FakeItEasy.sln"
 assembly_info   = "src/CommonAssemblyInfo.cs"
 version         = IO.read(assembly_info)[/AssemblyInformationalVersion\("([^"]+)"\)/, 1]
 version_suffix  = ENV["VERSION_SUFFIX"]
+repo_url        = "https://github.com/FakeItEasy/FakeItEasy"

```

Comment:
Good spot. I was hoping I'd get away with it. :wink: 

I thought about it, but it would mean constructing a URL from the scheme+host and the repo name. Seemed less hassle to duplicate.